cypress/support/api_commands.js

```diff
@@ -35,10 +35,88 @@ function reqLocalTransmitter() {
     headers: {
       accept: 'application/json',
     }
-  }).its('body').as('localTransmitter');
+  }).as('localTransmitter');
 }
 Cypress.Commands.add('reqLocalTransmitter', reqLocalTransmitter);
 
+function reqAllChannelsByTransmitter() {
+  return cy.request({
+    method: 'GET',
+    url: '/transmitters/local/channels',
+    headers: {
+      accept: 'application/json',
+    }
+  }).as('channels');
+}
+Cypress.Commands.add('reqAllChannelsByTransmitter', reqAllChannelsByTransmitter);
+
+function reqChannelBySlotTransmitter(slot) {
+  return cy.request({
+    method: 'GET',
+    url: '/transmitters/local/channels/' + slot,
+    headers: {
+      accept: 'application/json',
+    }
+  }).as('channel');
+}
+Cypress.Commands.add('reqChannelBySlotTransmitter', reqChannelBySlotTransmitter);
+
+function reqLatestTransmitterReading() {
+  return cy.request({
+    method: 'GET',
+    url: '/transmitters/local/latest_reading',
+    headers: {
+      accept: 'application/json',
+    }
+  }).as('latest_reading');
+}
+Cypress.Commands.add('reqLatestTransmitterReading', reqLatestTransmitterReading);
+
+function reqTransmitterReadingsByLatest(latest) {
+  return cy.request({
+    method: 'GET',
+    url: '/transmitters/local/readings?latest=' + latest,
+    headers: {
+      accept: 'application/json',
+    },
+  }).as('readings');
+}
+
+Cypress.Commands.add('reqTransmitterReadingsByLatest', reqTransmitterReadingsByLatest);
+
+function reqTransmitterReadingsByRange(start, end) {
+  return cy.request({
+    method: 'GET',
+    url: '/transmitters/local/readings?start=' + start + '&end=' + end,
+    headers: {
+      accept: 'application/json',
+    },
+  }).as('readings');
+}
+
+Cypress.Commands.add('reqTransmitterReadingsByRange', reqTransmitterReadingsByRange);
+
+function updateLocalTransmitterName(name, id, opts = {}) {
+  if (typeof id === 'undefined') {
+    id = Cypress.env('transmitter_id');
+  }
```

Comment:
I wouldn't pass the transmitter ID through the env, it should usually be 1 but it should be more reliable to use `'local'` as the default value; any other value for `id` and it would not be the 'local' transmitter so we can just call this `updateTransmitter()`  Also we're not just updating the name you're also blanking out the org and site.  
```suggestion
function updateTransmitter(name, id='local', opts = {}) {
```

Or would it work if you did this?
```suggestion
function updateTransmitter({name, org site}, id='local', opts = {}) {
  return cy.request({
    method: 'PUT',
    url: '/transmitters/' + id,
    body: {
      name,
      org,
      site,
    },
```
I don't recall if all three are required by the route or if it will only update the value(s) that are passed in the body.